features/google/src/views/ministories/singleDataStory.jsx

```diff
@@ -0,0 +1,12 @@
+import Story from "./story.jsx";
+
+export class SingleDataStory extends Story {
+    constructor(props, neededDataKey) {
+        super(props);
+        this._neededAnalyses = [neededDataKey];
```

Comment:
I don't get this very well. On one hand, it's plural and it's an array. On the other hand, it's a single key?

Reply:
It's an extension of Story which has only one key, as the logic is much simpler if that is the case I think

Reply:
Ah, OK, so this is not a new attribute. In that case, why it's not assigned with `super`?

Reply:
Bc it's children responsibility, aka the children need to put and it is used via super from them to here